tests/sanity/ignore-2.10.txt

```diff
@@ -0,0 +1,22 @@
+plugins/modules/import_eks.py import-2.6 # TODO issue #32
```

Comment:
For all the python version 2.x we can say `# Python 2.x is not supported`

Reply:
updated